aws-rds-customdbengineversion/aws-rds-customdbengineversion.json

```diff
@@ -76,6 +76,18 @@
       "type": "string",
       "description": "The ARN of the custom engine version."
     },
+    "SourceCustomDBEngineVersionIdentifier": {
+      "type": "string",
+      "description": "The identifier of the source custom engine version."
+    },
+    "UseAwsProvidedLatestImage": {
```

Comment:
This parameter may cause issues as it can be easily applied during the creation process, but problems may arise when modifying CEV. Customers will expect that updating their CFN stack with the CEV resource will trigger an update to keep their CEV up to date with the latest image at the time of the update. However, the truth is that this parameter will be ignored and the CEV will not be updated. On the other hand, if the customer makes any changes that require replacement and are not related to the image ID, such as changing the name of the CEV, the CEV image will be updated, resulting in inconsistency.

To resolve this issue, the CEV describe or RDS WS should provide an easy way to determine the latest image version. This will allow the UpdateHandler to determine if an upgrade is needed or not during invocation.